recipes/nsls2-collection/meta.yaml

```
@@ -0,0 +1,137 @@
+{% set name = "nsls2-collection" %}
+{% set year = "2021" %}
+{% set cycle = "2" %}
+{% set version = "0" %}
+{% set micro = "" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ year }}C{{ cycle }}.{{ version }}{{ micro }}
+
+build:
+  skip: true  # [py<37 or not linux]
+  number: 0
+
+requirements:
+  host:
+    - python
+  run:
+    - nsls2-analysis ==2021C2.0=*_1
```

This is to pick up the build number `0` for the `nsls2-analysis` recipe above.
```suggestion
    - nsls2-analysis ==2021C2.0=*_0
```